html/modules/custom/hr_paragraphs/src/Controller/ReliefwebController.php

```diff
@@ -12,7 +13,7 @@
 /**
  * Page controller for tabs.
  */
-class ReliefwebController extends ControllerBase {
+class ReliefwebController extends ControllerBase implements ContainerInjectionInterface {
```

Comment:
ControllerBase already implements `ContainerInjectionInterface` so this is not needed.